src/trpc/routers/calendarEventsRouter.ts

```diff
@@ -37,20 +37,23 @@ export const calendarEventsRouter = router({
             const { part, exceptionId, ...restInput } = input;
 
             if (exceptionId) {
-                return calendarEventMethods.updateEventException({
-                    part: 'exception',
-                    exceptionId,
-                    ...restInput,
-                });
+                return calendarEventMethods.updateEventException(
+                    {
+                        part: 'exception',
+                        exceptionId,
+                        ...restInput,
+                    },
+                    ctx.session.user,
+                );
             }
 
             switch (part) {
                 case 'exception':
-                    return calendarEventMethods.createEventException(input);
+                    return calendarEventMethods.createEventException(input, ctx.session.user);
                 case 'future':
-                    return calendarEventMethods.splitEventSeries(input, ctx.session.user.id);
+                    return calendarEventMethods.splitEventSeries(input, ctx.session.user);
                 case 'series':
-                    return calendarEventMethods.updateEventSeries(input);
+                    return calendarEventMethods.updateEventSeries(input, ctx.session.user);
```

Comment:
you probably want calendar event owner here, not the user?

Reply:
yes, you are right, it may be not the same user

Reply:
event creator can be null, so user here also needed